components/script/dom/globalscope.rs

```diff
@@ -2909,7 +2909,7 @@ impl GlobalScope {
     /// Returns a boolean indicating whether the event-loop
     /// where this global is running on can continue running JS.
     pub fn can_continue_running(&self) -> bool {
-        if self.downcast::<Window>().is_some() {
+        if !self.is::<Window>() {
```

Comment:
Does this reverse the logic of the original test? Previously it was true if `self` was a `Window`, but now it is false.

Reply:
I can simply remove the `!`(not) so it returns the right logic